cinema_service/settings.py

```diff
@@ -20,12 +21,10 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
```

Comment:
Security Issue: The secret key should not be committed to version control. Instead, you should use environment variables or a separate configuration file that is not committed to the repository. This is to avoid the secret key from being exposed to the public.